swift/logging/SwiftLogging.cpp

```diff
@@ -95,12 +98,14 @@ std::vector<std::string> Log::collectLevelRulesAndReturnProblems(const char* env
 void Log::configure() {
   // as we are configuring logging right now, we collect problems and log them at the end
   auto problems = collectLevelRulesAndReturnProblems("CODEQL_EXTRACTOR_SWIFT_LOG_LEVELS");
-  auto now = std::to_string(std::chrono::system_clock::now().time_since_epoch().count());
+  auto logBaseName = std::to_string(std::chrono::system_clock::now().time_since_epoch().count());
```

Comment:
`absl::StrCat`?

Reply:
I went on and even if not really related to this PR, I replaced all usages of `std::to_string` with `absl::StrCat` or `absl::StrAppend` :+1: